scheduler/src/cook/components.clj

```diff
@@ -69,7 +70,13 @@
                   (fn [] @mesos-pending-jobs-atom)
                   settings
                   (get-in mesos [:mesos-scheduler :leader-selector])
-                  mesos-leadership-atom))
+                  mesos-leadership-atom
+                  ;; TODO: Help? How do I go from config information (see example under cook.edn) into the arguments here?
```

Comment:
Here is where I'm looking for help.

Reply:
You can have a factory method in the rate limiter library and then invoke it using the config to get hold of the rate-limiter instance/function. Then you can pass the rate limiter as an argument in here.